act-rules-format.bs

```diff
@@ -451,63 +494,74 @@ Definitions {#definitions}
+    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requirements=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+  </dd>
+
+  <dt><dfn>Finding</dfn></dt>
+  <dd>
+    <p>When evaluating a [=test subject=] using a rule, the findings of that rule are:</p>
+    <ul>
+      <li>All [=test targets=] in the test subject</li>
+      <li>Which test targets met which [expectations](#expectations) in the test subject</li>
+      <li>The [=outcome=] (passed, failed or inapplicble) for the test subject</li>
+    </ul>
+  </dd>
+
+  <dt><dfn>Outcome</dfn></dt>
+  <dd>
+    <p>One of three types of conclusions that come from evaluating an ACT Rule on a [=test subject=]. An outcome can be:<p>
+    <ul>
+      <li>**Inapplicable:** No part of the test subject matches the applicability</li>
+      <li>**Passed:** The test subject meets all expectations</li>
+      <li>**Failed:** The test subject does not meet all expectations</li>
+    </ul>
+    <div class="note">
+      <p>**Note:** While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [=accessibility requirements=]. Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
+    </div>
+    <div class="note">
+      <p>**Note:** Implementers using the [[EARL10-Schema]] can express the outcome with the [outcome property](https://www.w3.org/TR/EARL10-Schema/#outcome). In addition to `passed` `failed` and `inapplicable`, EARL 1.0 also defined an `incomplete` outcome. While this can not be the outcome of an ACT Rule when applied in its entirety, it often happens that rules are only partially evaluated. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the `incomplete` outcome.
+    </div>
   </dd>
 
   <dt><dfn>Test Subject</dfn></dt>
   <dd>
     <p>A resource or collection of resources that can be evaluated by an ACT Rule.</p>
     <div class=example>
-      <p><strong>Example:</strong></p>
+      <p>**Example:**</p>
       <ul>
         <li>An HTML page, including all embedded scripts, style and images</li>
         <li>An EPUB document</li>
         <li>A Vue component file</li>
       </ul>
     </div>
     <div class=note>
-      <p><strong>Note</strong>: Implementers using the [[EARL10-Schema]] can express the test subject with the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
+      <p>**Note:**>: Implementers using the [[EARL10-Schema]] can express the test subject with the [subject property](https://www.w3.org/TR/EARL10-Schema/#subject)</p>
     </div>
   </dd>
 
   <dt><dfn>Test Target</dfn></dt>
   <dd>
     <p>A distinct part of the [=test subject=].</p>
```

Comment:
Should it say here that the test targets are the things that are applicable to the rule?
I know that it becomes a bit circular when following the link from the Applicability, but out of context, the definition seems very thin.